teamscale-client/build.gradle

```diff
@@ -12,6 +12,7 @@ publishAs {
 
 dependencies {
     api 'com.squareup.retrofit2:retrofit:2.9.0'
+    implementation 'com.squareup.okhttp3:okhttp:4.9.3'
```

Comment:
Was that the only necessary change to fix that? 😅

Reply:
The functionality was added somewhere around version 4.6 of okhttp. Retrofit is still using some 3.x version of okhttp by default. So after some research i found out that it should work with just adding the dependency and it worked.